src/main/resources/db/changelog/changes/06-create-users_roles-table.yaml

```
@@ -0,0 +1,23 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-users_roles-table
+      author: fedorovychh
+      changes:
+        - createTable:
+            tableName: users_roles
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: user_id
+                  type: bigint

```

It would be better to add constraints
```suggestion
                    nullable: false
                    references: users(id)
                    foreignKeyName: fk_user_role_user_id
                    deleteCascade: true
                    primaryKey: true
```